### PR TITLE
Add user stories for payload scheduler

### LINKED ISSUE #179: create user story for satisfying payload stakeholder needs
The stakeholder needs concerning payload (i.e., secondary & tertiary mission objectives) should map to one or more user stories

### PR DESCRIPTION
**Pull request by BhagatjiJimesh**
ODU and CGA Payload details
Closes #179

### CODE REVIEW

**Review by capsulecorplab — COMMENTED** [review]
TODO: add user story for (autonomous?) scheduled downlink

File: architecture/2-UserStories/5-PayloadScheduler.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+id: 5.0
+name: Payload Scheduler
+actor: Ground Station Operator 
+behavior: Schedule data recording at specfic time interval (or orbital location)
+rationale: Recording data for future transmission
+derivedFrom: []
+example: 
+- Ex1 Schedule data recording at 4min -10min (for a given orbital location) for ODU Payload?
+- Ex2 Schedule data recording at specfic time interval (or orbital location) for ODU Payload?
```
Comment (capsulecorplab) [inline architecture/2-UserStories/5-PayloadScheduler.yaml:9]: I'm thinking the second example should be a separate user story all together; i.e., separate scheduling by time interval from scheduling by orbital location

Comment (capsulecorplab) [reply architecture/2-UserStories/5-PayloadScheduler.yaml:9]: This also applies for `architecture/2-UserStories/5.1-ODUPayloadScheduler.yaml` and `architecture/2-UserStories/5.2-CGAPayloadScheduler.yaml`

**Review by capsulecorplab — CHANGES_REQUESTED** [review]
Need to add user stories for scheduling:
- [ ] telemetry downlink
- [ ]  eventlog downlink

**Comment by capsulecorplab** [comment]
Need to determine
- how much time to allocate per scheduled task
- how far into the future do we want to be able to schedule a task

**Comment by capsulecorplab** [comment]
dump health data info into eventlog per mission mode

File: architecture/2-UserStories/5.1-ODUPayloadSchedulerTLE.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+id: 5.1
+name: ODU Payload Scheduler
+actor: Ground Station Operator 
+behavior: Schedule data recording at specfic orbital location
+rationale: Recording data for future transmission
+derivedFrom: []
+example: 
+- Ex1 Schedule data recording at 4min -10min (for a given orbital location) for ODU Payload
```
Comment (capsulecorplab) [inline architecture/2-UserStories/5.1-ODUPayloadSchedulerTLE.yaml:8]: should remove this line

File: architecture/2-UserStories/5.2-CGAPayloadSchedulerTLE.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+id: 5.2
+name: CGA Payload Scheduler
+actor: Ground Station Operator 
+behavior: Schedule data recording at specfic orbital location
+rationale: Recording data for future transmission
+derivedFrom: []
+example: 
+- Ex1 Schedule data recording at 4min -10min (for a given orbital location) for CGA Payload
```
Comment (capsulecorplab) [inline architecture/2-UserStories/5.2-CGAPayloadSchedulerTLE.yaml:8]: should remove this line

File: architecture/2-UserStories/6-PayloadSchedulerTimeInterval.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+id: 5.0
```
Comment (capsulecorplab) [inline architecture/2-UserStories/6-PayloadSchedulerTimeInterval.yaml:1]: id s/b 6.0

File: architecture/2-UserStories/6.1-ODUPayloadSchedulerTimeInterval.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+id: 5.1
```
Comment (capsulecorplab) [inline architecture/2-UserStories/6.1-ODUPayloadSchedulerTimeInterval.yaml:1]: id s/b 6.1

File: architecture/2-UserStories/6.2-CGAPayloadSchedulerTimeInterval.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+id: 5.2
```
Comment (capsulecorplab) [inline architecture/2-UserStories/6.2-CGAPayloadSchedulerTimeInterval.yaml:1]: id s/b 6.2

File: architecture/2-UserStories/6.3-DownlinkPayloadDataTimeInterval.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+id: 5.3
```
Comment (capsulecorplab) [inline architecture/2-UserStories/6.3-DownlinkPayloadDataTimeInterval.yaml:1]: id s/b 6.3

**Review by capsulecorplab — CHANGES_REQUESTED** [review]
I would also remove `Ex1/Ex2` from the examples

**Review by capsulecorplab — CHANGES_REQUESTED** [review]
derivedFrom fields should point to their associated stakeholder needs

**Review by capsulecorplab — APPROVED** [review]
Will need to update examples with more concrete examples. Otherwise, LGTM

**Review by kchiu002 — APPROVED** [review]
LGTM

**Pull request merged**